tests/unit/adapters/tumblr-post-test.js

```diff
@@ -6,7 +7,44 @@ moduleFor('adapter:tumblr-post', 'Unit | Adapter | tumblr-post', {
 });
 
 // Replace this with your real tests.
-test('it exists', function(assert) {
-  var adapter = this.subject();
+test('it exists', function (assert) {
+  const adapter = this.subject();
   assert.ok(adapter);
 });
+
+test('pathForType', function (assert) {
+  const type = 'foo';
+  const adapter = this.subject({});
+
+  let result = adapter.pathForType();
+
+  assert.equal(result, 'posts', 'no type added if not defined');
+
+  adapter.set('type', type);
+
+  result = adapter.pathForType();
+  assert.equal(result, `posts/${type}`, 'type added to path');
+});
+
+test('namespace', function (assert) {
+  const blogUrl = 'foo.tumblr.com';
+  const adapter = this.subject({ blogUrl  });
+
+  assert.equal(adapter.get('namespace'), `v2/blog/${blogUrl}`, 'blogUrl added to api namespace');
+});
+
+test('ajaxOptions', function (assert) {
+  const apiKey = 'foo';
+  const adapter = this.subject({
+    apiKey
+  });
+  const hash = {};
+  const spy = Edgar.createSpy(adapter, '_super', hash);
+
+  const result = adapter.ajaxOptions();
+  const data = result.data;
```

Comment:
'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).